tests/test-throttle.py

```diff
@@ -546,15 +546,15 @@ class SlowPathIO(aioftp.PathIO):
 
     async def worker(fname):
 
-        client = aioftp.Client(loop=loop)
-        await client.connect("127.0.0.1", PORT)
-        await client.login()
-        await client.download(
+        _client = aioftp.Client(loop=loop, ssl=client.ssl)
```

Comment:
There's no point/value in underscoring bunch of vars.

Reply:
so how would you like to call a variable?

Reply:
keep it as is, I didn't realize that it's an "extended boolean"

Reply:
Oh, that was not about ssl